docs/examples/general/general_ops_index.py

```diff
@@ -2,11 +2,26 @@
     underline_char="=",
     entries=[
         "expressions/index.py",
-        doc_entry("reductions.ipynb",
-                  op_reference('fn.reductions', "Tutorial describing how to use reductions")),
-        "tensor_join.ipynb",
-        "reinterpret.ipynb",
-        "normalize.ipynb",
-        "../math/geometric_transforms.ipynb",
-        "erase.ipynb",
+        doc_entry(
+            "reductions.ipynb",
+            op_reference('fn.reductions', "Tutorial describing how to use reductions")),    
+        doc_entry(
+            "tensor_join.ipynb",
+            op_reference('fn.cat', "Tutorial describing tensor joining")),
+        doc_entry(
+            "reinterpret.ipynb",
```

Comment:
I think you should add `squeeze`, `expand_dims` and `reinterpret` as this tutorial describes them as well.

Reply:
Done